steps/yarn/0.0.2/step.yml

```diff
@@ -0,0 +1,37 @@
+title: Run yarn command
+summary: Runs `yarn` with the given command and args.
+description: Runs `yarn` with the given command and args.
+website: https://github.com/jonoirwinrsa/steps-yarn
+source_code_url: https://github.com/jonoirwinrsa/steps-yarn
+support_url: https://github.com/jonoirwinrsa/steps-yarn/issues
+published_at: 2016-10-20T13:25:54.991730369+02:00
+source:
+  git: https://github.com/jonoirwinrsa/steps-yarn.git
+  commit: 9b65131340ace1399bee8b2916eb2d65b5714db7
+host_os_tags:
+- osx-10.10
+project_type_tags:
+- react-native
```

Comment:
remove this or change to `ios, android`, or else the step won't show up by default on bitrise.io only when you uncheck the "filter steps by platform" in the step list